server/base/src/test/java/org/apache/accumulo/server/conf/NamespaceConfigurationTest.java

```diff
@@ -81,6 +81,8 @@ public void setUp() {
         Map.of(Property.INSTANCE_SECRET.getKey(), "sekrit"))).anyTimes();
     propStore.registerAsListener(eq(nsPropStoreKey), anyObject());
     expectLastCall().anyTimes();
+    propStore.invalidate(nsPropStoreKey);
+    expectLastCall().anyTimes();
```

Comment:
I've found that `atLeastOnce()` is usually what we want when we use `anyTimes()`

```suggestion
    expectLastCall().atLeastOnce();
```